whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -276,7 +276,8 @@ private Response getResponse(HttpServletRequest request, List<RpslObject> result
                         getRequestUrl(request),
                         resultObject,
                         objectDao.getLastUpdated(resultObject.getObjectId()),
-                        abuseCFinder.getAbuseContactRole(resultObject)))
+                        abuseCFinder.getAbuseContactRole(resultObject),
```

Comment:
Do we need two separate calls to the abuseCFinder? can you read the abuse-mailbox from the RpslObject from the  getAbuseContactRole() ?

Reply:
I've added the abuse role object to the AbuseContact so we don't need multiple lookups and we have all the information we need in one place.